### PR TITLE
mdbx_env_warmup initial commit

### PR DESCRIPTION
**Pull request by racytech**

### CODE REVIEW

File: mdbx/mdbxgo.c [hunk 1]
```diff
@@ -113,6 +113,6 @@ int mdbxgo_dcmp(MDBX_txn *txn, MDBX_dbi dbi, char *adata, size_t an, char *bdata
     return mdbx_dcmp(txn, dbi, &a, &b);
 }
 
-
-
-
```
Comment (AskAlexSharov) [inline mdbx/mdbxgo.c:118]: Please take a look if we really need this wrapper. Other wrappers was created to reduce amount of CGO allocations. But allocations amount here are not important and seems func doesn’t do anything. Can call mdbx’s func directly.

**Review by AskAlexSharov — CHANGES_REQUESTED** [review]

File: mdbx/mdbxgo.c (hunk 1 above)
Comment (racytech) [reply mdbx/mdbxgo.c:118]: I removed the wrapper. I am thinking if `MdbxEnvWarmup` has to be struct method?

Comment (AskAlexSharov) [reply mdbx/mdbxgo.c:118]: Yes. Seems method of Txn 
Can’t pass nil txn to this func?

**Review by AskAlexSharov — APPROVED** [review]

**Comment by AskAlexSharov** [comment]
oh, seems we forgot to auto-convert `time.Duration` to `timeout_seconds_16dot16`. i will do

**Pull request merged**